File: module-git/src/main/java/com/timecat/module/git/sgit/activities/delegate/actions/AddAllAction.java
package com.timecat.module.git.sgit.activities.delegate.actions;

import com.timecat.module.git.sgit.activities.RepoDetailActivity;
import com.timecat.module.git.sgit.database.models.Repo;
import com.timecat.module.git.sgit.repo.tasks.repo.AddToStageTask;

public class AddAllAction extends RepoAction {

    public AddAllAction(Repo repo, RepoDetailActivity activity) {
        super(repo, activity);
    }

    @Override
    public void execute() {
        AddToStageTask addTask = new AddToStageTask(mRepo, ".");
        addTask.executeTask();
        mActivity.closeOperationDrawer();
    }

}

File: module-git/src/main/java/com/timecat/module/git/sgit/activities/delegate/actions/AddAllAction.kt
package com.timecat.module.git.sgit.activities.delegate.actions

import com.timecat.module.git.sgit.activities.RepoDetailActivity
import com.timecat.module.git.sgit.database.models.Repo
import com.timecat.module.git.tasks.AddToStageTask

class AddAllAction(repo: Repo, activity: RepoDetailActivity) : RepoAction(repo, activity) {
    override fun execute() {
        val addTask = AddToStageTask(mRepo, ".")
        addTask.executeTask()
    }
}